### PR TITLE
First pass at removing core profile requirements, #413

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: starksm64 <starksm64@gmail.com>

### CODE REVIEW

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 1]
```diff
@@ -9,6 +9,8 @@ The Jakarta EE Web Profile Specification,
 published in conjunction with the present specification, defines the
 first Jakarta EE profile, the Web Profile.
 
+The Jakarta EE Core Profile Specification is a minimal profile that is published separately from the Web Profile and this specification.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc:12]: Just to be clear, should you say "..separately from the Web Profile and this **Platform** specification."

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 2]
```diff
@@ -147,7 +149,7 @@ annotations defined by the Common Annotations specification ( _Resource_,
  _Resources_, _PostConstruct_, _PreDestroy_ )
 
 The following functionality is required to be
-supported in all Jakarta EE profiles:
+supported in all Jakarta EE profiles other than the Core Profile:
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc:152]: I think this whole section "Requirements for All Jakarta EE Profiles" needs to be re-worked.  Just inserting the "other than Core Profile" phrase opens the door for other Profiles to also not follow what's listed in this section.  Maybe this section just only contain the Java SE requirements and Common Annotations?  And, the requirement for Transactions and java: naming context should be moved to the Web Profile?  Just an idea.  But, we need to do something more than just inserting the "other than Core Profile" phrase.

Comment (starksm64) [reply specification/src/main/asciidoc/platform/Profiles.adoc:152]: The simplest would be to remove JNDI/JTA from this section and add it as required in the Web Profile.

File: specification/src/main/asciidoc/platform/Profiles.adoc (hunk 1 above)
Comment (starksm64) [reply specification/src/main/asciidoc/platform/Profiles.adoc]: Fixed

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 3]
```diff
@@ -136,7 +138,7 @@ requirements.
 [[a3240]]
 === Requirements for All Jakarta EE Profiles
 
-The Java Platform, Standard Edition 8 is the
+The Java Platform, Standard Edition 11 is the
 required API foundation for any Jakarta EE profile.
```
Comment (ederks85) [inline specification/src/main/asciidoc/platform/Profiles.adoc:142]: I would replace the word "foundation" with "compile level". Foundation feels a bit too open for interpretation, where one could read it as compile and runtime level.

**Review by ederks85 — APPROVED** [review]

File: specification/pom.xml [hunk 4]
```diff
@@ -205,44 +205,6 @@
                             </attributes>
                         </configuration>
                     </execution>
-                    <!-- ManagedBeans HTML generation -->
```
Comment (scottmarlow) [inline specification/pom.xml:208]: I assume this is for moving managed beans to the core profile.

**Review by scottmarlow — APPROVED** [review]

File: specification/pom.xml (hunk 4 above)
Comment (starksm64) [reply specification/pom.xml:208]: The managed beans spec was moved to its own repo:
https://github.com/eclipse-ee4j/managed-beans-spec

It was deprecated for EE10 as I remember, so I'm not sure what we are doing with that spec.

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**